inc/Base/Enqueue.php

```diff
@@ -89,4 +91,28 @@ public function enqueue_front_scripts() {
 		wp_enqueue_style( 'smailypluginstyle', SMAILY_PLUGIN_URL . 'static/front-style.css', array(), SMAILY_PLUGIN_VERSION );
 	}
 
+	/**
+	 * Dequeues all 3rd party styles on Smaily module settings page.
+	 *
+	 * @return void
+	 */
+	public function dequeue_admin_styles() {
+
+		if ( get_current_screen()->base !== 'toplevel_page_smaily-settings' ) {
+			return;
+		}
+
+		global $wp_styles;
+		foreach ( $wp_styles->queue as $style_handle ) {
+			if ( $style_handle === 'smailypluginstyle' ) {
+				continue;
+			}
+			$style_src_path  = $wp_styles->registered[ $style_handle ]->src;
+			$plugins_dir_url = content_url( 'plugins' );
```

Comment:
You can move this before the loop, because `content_url( 'plugins' )` is always returning the same result. It will save some CPU cycles if installation should have a lot of plugins applying styles.

Reply:
Added a6f70ae